src/App.js

```diff
@@ -66,8 +114,9 @@ const ZshellDiv = styled.div`
   box-shadow: 0 10px 8px #888;
 `;
 
-const StyledSection = styled.section`
-  max-width: 50vw;
+const StyledDiv = styled.div`
+  width: 75vw;
```

Comment:
I'm not sure what you mean by this: choose between `width:` and `max-width:`? this would make make it less responsive. Or between the `max-width:` i used before and the one now? Sorry, but i really specaluted over this for a while now

Reply:
Hi @GeorgJuranek ✌️
I would recommend you to set the `max-with` to the `<main>` element in your global styles and to remove the `max-with` property from your child component. This way you can better control behavior of the child elements. You can easily test it in the dev tools of your browser. 

And of course you can decide whether you want to implement my proposal or not. 🙂

Example:
```css
main {
/.../
max-with: 500px;
margin: 0 auto;
/.../
}
```